lib/modules/platform/bitbucket-server/index.spec.ts

```diff
@@ -1859,6 +1860,75 @@ describe('modules/platform/bitbucket-server/index', () => {
           });
           expect(pr?.number).toBe(5);
         });
+
+        it('should use platform automerge', async () => {
+          const scope = await initRepo();
+          scope
+            .post(
+              `${urlPath}/rest/api/1.0/projects/SOME/repos/repo/pull-requests`,
+            )
+            .reply(200, prMock(url, 'SOME', 'repo'))
+            .get(
+              `${urlPath}/rest/api/1.0/projects/SOME/repos/repo/pull-requests?state=ALL&limit=100&role.1=AUTHOR&username.1=abc`,
+            )
+            .reply(200, {
+              isLastPage: true,
+              values: [],
+            })
+            .post(
+              `${urlPath}/rest/api/1.0/projects/SOME/repos/repo/pull-requests/5/merge?version=1`,
+            )
+            .reply(200);
+
+          // Simulate version >= 8.15.0
+          vi.spyOn(semver, 'lt').mockReturnValue(false);
```

Comment:
we prefer to use a special initPlatform call with a different mock version. see other tests / other platforms how to do it